src/javascript/app/pages/trade/purchase.js

```diff
@@ -62,23 +64,31 @@ const Purchase = (() => {
         if (error) {
             container.style.display = 'block';
             message_container.hide();
-            confirmation_error.show();
-            let message = error.message;
-            if (/RestrictedCountry/.test(error.code)) {
-                let additional_message = '';
-                if (/FinancialBinaries/.test(error.code)) {
-                    additional_message = localize('Try our [_1]Volatility Indices[_2].', [`<a href="${urlFor('get-started/binary-options', 'anchor=volatility-indices#range-of-markets')}" >`, '</a>']);
-                } else if (/Random/.test(error.code)) {
-                    additional_message = localize('Try our other markets.');
+            if (/AuthorizationRequired/.test(error.code)) {
+                $(authorization_error).setVisibility(1);
```

Comment:
no need for jquery wrapper, we have a setVisibility prototype function declared for DOM